docs/docs/emotion.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Emotion
+---
+
+In this guide, we'll walk through setting up a site with the CSS-in-JS library [Emotion](https://emotion.sh).
+
+Emotion is a performant and flexible CSS-in-JS library. Building on many other CSS-in-JS libraries, it allows you to style apps quickly with string or object styles. It has predictable composition to avoid specificity issues with CSS. With source maps and labels, Emotion has a great developer experience and great performance with heavy caching in production.
+
+First, open a new terminal window and run the following to create a new site:
+
+```shell
+gatsby new emotion-tutorial https://github.com/gatsbyjs/gatsby-starter-hello-world
```

Comment:
```suggestion
npx gatsby new emotion-tutorial https://github.com/gatsbyjs/gatsby-starter-hello-world
```

(No way you could have known this; I need to update some documentation to reference moving off of the globally installed version!)